Z/ZipFlow/bundled/patches/windows.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/zipflow.c b/zipflow.c
+index 30d4bd4..a6ddaf6 100644
```

Comment:
Could you please upstream also this patch?

Reply:
Done